DIPs/DIP1028.md

```diff
@@ -50,11 +54,42 @@ compiler switch:
 
 There are no grammar changes.
 
+### Non-Virtual Templated Functions
+
+Non-virtual Function templates get their attributes inferred if they are not explicitly marked.
+Hence, this proposal should not affect them.
+
 ## Breaking Changes and Deprecations
 
 This will likely break most code that has not already been annotated with `@safe`,
-`@trusted`, or `@system`. Fortunately, the solution is easy, although tedious: annotate
-functions that aren't safe with `@trusted` or `@system`.
+`@trusted`, or `@system`. Annotate functions that aren't safe with `@system`.
+Once the project compiles again successfully, start with the leaves market `@system` and
+modify as required to make them `@safe`.
+
+In general, `@system` should not be applied to blocks of functions. It should be added
+specifically to each function that requires it.
+
+### Extern C and C++ Functions
+
+An unmarked `extern (D)` function will now be considered `@safe`. If its implementation is not recompiled,
+it will no longer link because the mangling will be different. But for `extern (C)` and `(C++)` functions,
+safety is not part of the mangling and it will compile and link without error. This has always
+relied on the user getting it correct.
+
+Interfaces to extern C and C++ functions should always be explicitly marked.
+
+### 3rd Party Libraries
+
+It's best practice for 3rd party libraries to use explicit safety annotations and not rely on defaults.
```

Comment:
Why is this a "best practice"? Not relying on defaults is only an issue if the default changes, do you plan on changing the defaults frequently? Otherwise if this is a best practice for some reason, why simply change the default? This DIP should instead require a user to annotate their functions appropriately.